proposals/NNNN-importing-forward-declared-objc-interfaces-and-protocols.md

```diff
@@ -0,0 +1,177 @@
+# Importing Forward Declared Objective-C Interfaces and Protocols
+
+* Proposal: [SE-NNNN](NNNN-importing-forward-declared-objc-interfaces-and-protocols.md)
+* Authors: [Nuri Amari](https://github.com/NuriAmari)
+* Review Manager: TBD
+* Implementation: https://github.com/apple/swift/pull/61606
+
+## Introduction
+
+This proposal seeks to improve the usability of existing Objective-C libraries from Swift by reducing the negative
+impact forward declarations have on API visibility from Swift. We wish to start synthesizing placeholder types to
+represent forward declared Objective-C interfaces and protocols in Swift.
+
+Swift-evolution thread: [Discussion thread topic for that proposal](https://forums.swift.org/t/pitch-importing-forward-declared-objective-c-classes-and-protocols/61926)
+
+## Motivation
+
+Forward declarations are very common in many existing Objective-C code bases, used often to break cyclic dependencies or to improve build performance.
+Unfortunately, when it comes to "importability" into Swift they are quite detrimental.
+
+As it stands, the ClangImporter will fail to import any declaration that references a forward declared type in many common cases. This means a single
```

Comment:
I think we can be more concise here and get straight to the point:

```suggestion
Unfortunately, they can cause interoperability problems for Swift. ClangImporter will silently drop declarations that reference a forward declared type in many common cases. This means a single
```

I suggest the wording "silently drop" vs. "fail to import" since "fail" could be interpreted to mean that the compiler fails or emits an error, which isn't the case here; it silently continues and the user is none the wiser.